pandas/tests/io/test_stata.py

```diff
@@ -589,6 +589,15 @@ def test_105(self):
         df0['psch_dis'] = df0["psch_dis"].astype(np.float32)
         tm.assert_frame_equal(df.head(3), df0)
 
+    def test_value_labels_old_format(self):
+        # GH 19417
+        #
+        # Test that value_labels() returns an empty dict if the file format
+        # predates supporting value labels.
+        dpath = os.path.join(self.dirpath, 'S4_EDUC1.dta')
+        reader = StataReader(dpath)
```

Comment:
need to close the reader (or do this in a context manager) see other tests

Reply:
`close()`d